### PR TITLE
59 delete webdriverio adapter

### PR DESCRIPTION
**Pull request by adela-istrate-hs**

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs @valentin-radulescu-hs @flavius-tirnacop-hs Can you review?

File: index.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
 var Mugshot = require('./lib/mugshot.js');
 
 Mugshot.adapters = {
-  WebdriverIO: require('./lib/adapters/webdriverio.js'),
   LooksSame: require('./lib/adapters/looks-same.js')
 };
 
 Mugshot.interfaces = {
-  browser: require('./lib/interfaces/browser.js'),
```
Comment (andrei-picus-hs) [inline index.js]: See https://github.com/uberVU/mugshot/pull/62/files#r72799123.

**Comment by flavius-tirnacop-hs** [comment]
LGTM

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs Can I merge this?

File: lib/interfaces/browser.js [hunk 2]
```diff
@@ -1,47 +1,61 @@
-/* eslint-disable no-unused-vars */
+var Browser = require('../interfaces/browser.js');
```
Comment (andrei-picus-hs) [inline lib/interfaces/browser.js]: What happened here?

**Comment by andrei-picus-hs** [comment]
Maybe we should have an end to end test that requires Mugshot from `index.js`. It would've caught the broken browser interface. Someone please create an issue for this.

File: lib/interfaces/browser.js (hunk 2 above)
Comment (adela-istrate-hs) [inline lib/interfaces/browser.js]: I don't know. I will fix it.

**Comment by adela-istrate-hs** [comment]
I created an issue and I fixed the broken interface. Can I merge now?

**Comment by andrei-picus-hs** [comment]
Not yet 😄 

image

File: lib/interfaces/browser.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (andrei-picus-hs) [inline lib/interfaces/browser.js]: Remove.

**Pull request merged**